python/GangaGaudi/Lib/Applications/GaudiBase.py

```diff
@@ -190,7 +190,7 @@ def cmt(self, command):
                 env=self.getenv(False),
                 cwd=self.user_release_area)
 
-    def unprepare(self):
+    def unprepare(self, force=False):
```

Comment:
I think the calls to move the counter up and down are still needed from memory but the interface should be respected yes so i think this is the minimal change needed.